### DISCUSSION TITLE
I'd like to access documentation more quickly in the Editor

### DISCUSSION THREAD

**Discussion by StachuDotNet** [darklang/dark#4319] [Feedback]
I often head to docs.darklang.com to check for the [keyboard shortcuts](https://docs.darklang.com/reference/keyboard-mapping) and [the full list of fns available](https://ops-documentation.builtwithdark.com/?pretty=1).

I'd like to instead reference these within the Editor.

**Comment by StachuDotNet** [comment]
My immediate thought is that I'd like the "Docs" button currently in the bottom right of the canvas to instead be a "docs" menu at the bottom of the left sidebar. Under that menu, I could access these things quickly, and have a general link to docs.darklang.com for other uses.

**Reply by StachuDotNet** [reply]
Something I really like about dynalist.io (see example doc [here](https://dynalist.io/d/DApXWd0Y4xEIc1aymL5FG2UC)) is that I can hit `ctrl+?` at any point to see the full list of keyboard shortcuts.

**Comment by StachuDotNet** [comment]
I could see keyboard shortcuts shown throughout the Editor in a more nuanced way (i.e. showing only what's relevant in the given context), but an over-all view has value as well.

**Reply by pbiggar** [reply]
My thoughts here, which I still wanna play with, are to show available keys at any point in a small menu at the bottom (toggleable of course). I also want to try out an interface similar to spacemacs where you press the leader key and then are given a menu of possible keys to press next.

**Comment by StachuDotNet** [comment]
as for the list of available fns, I could see a view similar to the current Package list in the sidebar, just above i, focused on StdLib fns.
